bindings/cs/rl.net.cli.test/UnicodeTest.cs

```diff
@@ -554,16 +554,6 @@ void RegisterNativeOverride(Func<IntPtr, IntPtr> nativeOverrideCallback)
             Run_StringReturnMarshallingTest<RankingResponse>(modelIdToReturn, RegisterNativeOverride, rankingResponse => rankingResponse.ModelId, nameof(NativeMethods.GetRankingModelId));
         }
 
-        private void Run_GetRankingEventId_Test(string eventIdToReturn)
```

Comment:
You should bring this back. It is EventId vs. ModelId test.

Reply:
actually i deleted the wrong one :)